### PR TITLE
Add SETI partner feature, fix YouTube embed for offline first

### PR DESCRIPTION
**Pull request by mikebroberg**

### CODE REVIEW

File: _data/projects.yml [hunk 1]
```diff
@@ -522,14 +522,16 @@
     - Scala
 
 - title: "SETI@IBMCloud: SETI data, publicly available, from IBM"
+  features:
+    - seti
   headline: "Raw data from the SETI Institute for you to crunch in the search for extraterrestrial life."
   links: 
     GitHub: "https://github.com/ibm-watson-data-lab/seti_at_ibm"
     Medium: "https://medium.com/ibm-watson-data-lab/simulating-e-t-e34f4fa7a4f0"
```
Comment (gadamc) [inline _data/projects.yml:530]: Hmm... this Medium article isn't technically part of SETI@IBMCloud. It's part of the ML4SETI code challenge. Those are two separate things.

File: _features/seti.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+<script src="https://gist.github.com/gadamc/6284fe9b0cb42d8af0c0943f02b2c1cd.js"></script>
+
+ibmos2spark comes pre-installed on the IBM Data Science Experience, with a UI feature for inserting credentials to code for faster data ingestion. The package works for these languages:
+
+{:.browser-default}
+- Python
+- Scala
+- R, sparklyR (support for IBM Cloud Object Storage coming soon)
+- R, sparkR (support for IBM Cloud Object Storage coming soon)
```
Comment (gadamc) [inline _features/seti.md]: can remove the "support for IBM Cloud Object Storage coming soon"... it's already there.

File: _data/projects.yml (hunk 1 above)
Comment (mikebroberg) [reply _data/projects.yml:530]: I see your point, but I think one has to be a subset of the other. So if you want that thing to be ml4seti, then that's fine. We can change the name.

File: _features/seti.md (hunk 2 above)
Comment (mikebroberg) [reply _features/seti.md]: OK done.

File: _data/projects.yml [hunk 3]
```diff
@@ -522,14 +522,16 @@
...
     dataset: "https://github.com/ibm-watson-data-lab/seti_at_ibm/blob/master/notebooks/all_signaldb.ipynb"
     developerWorks: "https://developer.ibm.com/clouddataservices/2016/09/29/seti-data-on-ibm-cloud/"
```
Comment (gadamc) [inline _data/projects.yml:532]: The developerWorks is focused on the Hackathon that was part of the ML4SETI code challenge... So, it's not in the SETI@IBMCloud section as well.

**Pull request merged**